testcases/network/tcp_cmds/ssh/ssh_setup

```diff
@@ -35,6 +35,18 @@ do_setup()
 	fi
 
 	echo "$TEST_USER:$TEST_USER_PASSWD" | chpasswd
+	if [ $? -ne 0 ]; then
```

Comment:
seems like you should just install `chpasswd` and we should document it as a requirement for ssh